### ISSUE TITLE
Add a confirm delete dropdown to delete overrides

### ISSUE THREAD

**Issue by harshsbhat** [unkeyed/unkey#2245]
### Preliminary Checks

- [X] I have reviewed https://unkey.com/docs for existing features that would solve my problem

- [X] I have searched for existing feature requests: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a feature request directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Is your feature request related to a problem? Please describe.

Currently, if you try to delete overrides. It is immediately deleted. This might not be ideal as someone might miss-click it and have their override gone within a second.  

https://app.unkey.com/ratelimits/namespaceId/overrides/

### Describe the solution

Add a confirm modal like every other delete button to ask the user whether they want to delete their override before actually deleting it.

**Comment by chronark** [comment]
I'm not sure here, it's not a critical piece, and deleting it on accident doesn't have many consequences.

**Comment by harshsbhat** [comment]
Yeah, maybe not a written modal like others. But a click-based confirm modal should do right?

**Comment by pragyananda** [comment]
i am working on it .

**Comment by harshsbhat** [comment]
> i am working on it .

its not approved

**Comment by pragyananda** [comment]
ok, no problem if this is approved then please assign me.

**Comment by chronark** [comment]
> Yeah, maybe not a written modal like others. But a click-based confirm modal should do right?

yeah, I was thinking this too
maybe turning the button into a dropdown with a single "confirm delete" action

**Comment by harshsbhat** [comment]
> > Yeah, maybe not a written modal like others. But a click-based confirm modal should do right?
> 
> yeah, I was thinking this too maybe turning the button into a dropdown with a single "confirm delete" action

Sounds good to me

**Comment by Aryant01** [comment]
can you assign me this issue to me

**Comment by harshsbhat** [comment]
@chronark don't want to sound like this. But this should award me 50-150 points right?
